task/net-api/src/lib.rs

```diff
@@ -265,5 +285,30 @@ impl From<Ipv6Address> for smoltcp::wire::Ipv6Address {
     }
 }
 
+/// Upstream SP port
+///
+/// Values are based on the KSZ8463's numbering (1-3); port 3 is connected to
+/// the SP itself and cannot be used as a source / destination.
+#[derive(Copy, Clone, Debug, PartialEq, Eq)]
+pub enum SpPort {
+    One,
+    Two,
+}
+
+/// Configuration for a SP VLAN
+#[derive(Copy, Clone, Debug)]
```

Comment:
again with the deriving `Debug`, always with the `Debug`... :)

Reply:
this is my emotional support `#[derive(Debug)]`, it's allowed to go into the embedded system